internal/pal/intialize.go

```diff
@@ -8,9 +8,10 @@ func init() {
 		if err != nil {
 			panic(err)
 		}
-	})
+	}, "\n")
```

Comment:
>\n") [](http://example.com/codeflow?start=5&length=4)

might Windows users want \r\n ?

Reply:
Added a defaultLineBreak function that returns \n or \r\n depending on platform